dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -410,13 +410,14 @@ public Token addGoogleToken(@ApiParam(hidden = true) @Auth Optional<User> authUs
     }
 
     private void acceptTOSAndPrivacyPolicy(User user) {
+        Date date = new Date();
         if (user.getTOSVersion() != CURRENT_TOS_VERSION) {
             user.setTOSVersion(CURRENT_TOS_VERSION);
-            user.setTOSVersionAcceptanceDate(new Date());
+            user.setTOSVersionAcceptanceDate((date));
```

Comment:
Not a big deal, but I actually prefer what it was before due to this https://refactoring.com/catalog/reduceScopeOfVariable.html kind of thing. 

i.e. anonymous variable with no scope better than local scope better than larger scope

Reply:
Yes, good point, it was at the back of my mind as well. I had been going to suggest doing a test at the top for if either condition was not true, but then that seemed like overkill, as you would then have to test each each condition individually again.

I do think it would be weird to have two dates off my milliseconds or less when they should be the same, and that bothered me slightly more. :)